### PR TITLE
[Web][SIMS #1170] Change date format

### PR DESCRIPTION
**Pull request by henryoforeh-aot**
- Replaced all places where dateString is used with dateOnlyLongString.
- Removed dateString function from useFormatters.
- Minor fix on dateOnlyLongString function to align with date format.

### CODE REVIEW

File: sources/packages/web/src/composables/useFormatters.ts [hunk 1]
```diff
@@ -14,38 +14,14 @@ export const DATE_ONLY_ISO_FORMAT = "YYYY-MM-DD";
  */
 
 export function useFormatters() {
-  /**
-   * Get the date only part of a date/time or string.
-   * @param date date/time or string to have the date extracted.
-   * @returns date only string in ISO format YYYY-MM-DD.
-   */
-  const getISODateOnlyString = (date: Date | string): string => {
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/composables/useFormatters.ts:22]: `getISODateOnlyString` should not be removed. It should not be used to display dates on UI but is can be used to send date-only values to the API.

Comment (henryoforeh-aot) [reply sources/packages/web/src/composables/useFormatters.ts:22]: I will put it back.

File: sources/packages/web/src/views/aest/student/SINManagement.vue [hunk 2]
```diff
@@ -198,7 +198,7 @@ export default {
         processingEditExpiryDate.value = true;
         const formioForm =
           modalResult as FormIOForm<UpdateSINValidationAPIInDTO>;
-        formioForm.data.expiryDate = getISODateOnlyString(
+        formioForm.data.expiryDate = dateOnlyLongString(
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/views/aest/student/SINManagement.vue:201]: Please revert this change. This is formatting the date to be sent to the API, not to the UI.

Comment (henryoforeh-aot) [reply sources/packages/web/src/views/aest/student/SINManagement.vue:201]: Noted

**Review by andrewsignori-aot — COMMENTED** [review]
Nice work, please take a look at the comments.
Was the API reviewed also? I believe that there are some dates coming formatted from the API also.

File: sources/packages/web/src/components/students/ApplicationDetails.vue [hunk 3]
```diff
@@ -32,6 +32,8 @@ import {
   StudentApplicationDetailsForTracking,
 } from "@/types";
 
+import { useFormatters } from "@/composables";
```
[the remark this answers is not in the document]
Comment (henryoforeh-aot) [reply sources/packages/web/src/components/students/ApplicationDetails.vue:34]: Removed.

File: sources/packages/web/src/composables/useFormatters.ts (hunk 1 above)
Comment (henryoforeh-aot) [reply sources/packages/web/src/composables/useFormatters.ts:22]: I have put it back.

Comment (guru-aot) [inline sources/packages/web/src/composables/useFormatters.ts:22]: i think this should not be deleted. As its used in many references of controllers.

Comment (henryoforeh-aot) [reply sources/packages/web/src/composables/useFormatters.ts:22]: I have put it back.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the cleanups and extending the changes to the API, look good 👍

File: sources/packages/api/src/utilities/date-utils.ts [hunk 4]
```diff
@@ -165,28 +141,12 @@ export function getExtendedDateFormat(date: Date): string {
 /**
  * Get the extended date format(2021 Mar 22) for the date given
  * @param date date to be retrieved as Extended date format
- * @returns extended date format like March, 22 2021
+ * @returns extended date format like Mar 22 2021
  */
-export function getDateOnlyFormat(date: Date): string {
+export function getDateOnlyFormat(date?: string | Date): string {
```
Comment (guru-aot) [inline sources/packages/api/src/utilities/date-utils.ts:146]: 👍

**Review by guru-aot — APPROVED** [review]
LGTM, Nice work @henryoforeh-aot . Thanks for doing the changes.

**Review by ann-aot — APPROVED** [review]
👍

**Review by dheepak-aot — APPROVED** [review]
LGTM @henryoforeh-aot 👍
